packages/suite/src/components/suite/Modal/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Modal as TrezorModal, ModalProps } from '@trezor/components';
+import { DESKTOP_WRAPPER_BORDER_WIDTH } from '@suite-constants/layout';
+import { isDesktop, isMac } from '@suite-utils/env';
+
+const Modal = (props: ModalProps) => (
+    <TrezorModal
+        {...{
+            ...props,
+            ...(isDesktop() && !isMac() && { desktopBorder: DESKTOP_WRAPPER_BORDER_WIDTH }),
```

Comment:
couldn't it be simplified to this?
```
<TrezorModal 
    {...props} 
    desktopBorder={isDesktop() && !isMac() ? DESKTOP_WRAPPER_BORDER_WIDTH : undefined}
/>
```
I don't know why but the double spread seems weird to me :D

Reply:
@tomexx what do you think? 😁